### DISCUSSION TITLE
Auto Minify & View Leaderboard Code

### DISCUSSION THREAD

**Discussion by LaCocoRoco** [cssbattle/forum#196] [Q&A] [answered]
Hi, while watching a youtube video of jessica chan i stumbled over cssbattle.
I think it is a realy good way to learn and play, but i have two things i am probably missing.
First: Is there a way to "Autominify" the code, because i do not want to waste readability for the sake of better scoring.
Second: Can i somehow watch the solution of the leaderbord? A good way to get better, is to see how professinel fix their problems.

Yours Andreas

**Comment by rafehimself** [comment]
for 2 nope, only in open targets, other than that ppl keep them rather secret 🙄  you can find average scores in google

**Comment by gunnarbittersmann** [comment, marked as answer]
With a pro account, you can use plug-ins in the code editor field. You could write a plug-in to remove space. Not the easiest task, though: in `margin: 10 31`, you can remove the first space, but not the second.
Also, the execution of a plug-in changes the code not just for submission, but in the editor.
So the answer to “autominify” is probably “no.”

**Comment by LaCocoRoco** [comment]
Thanks for the help! I could only mark one answer as correct, so i marked the later one.
